meta/parse.pl

```diff
@@ -2056,6 +2056,8 @@ sub ProcessIsAclField
 
     return "true" if $attr =~ /^SAI_ACL_ENTRY_ATTR_(USER_DEFINED_)?FIELD_\w+$/;
 
+    return "true" if $attr =~ /^SAI_UDF_MATCH_ATTR_\w+_TYPE$/;
```

Comment:
this is optimistic, since currently all TYPE ending attr are alc_field_data, but later on this may not be the case, but maybe this would be catched by current saisanitycheck, but if not then attributes would need to be typed here explicitly

Reply:
Yes, in check_attr_acl_field_or_action(), the TYPE ending attr are checked. If new TYPE ending attr is added as not aclfield data, the check will be failed. Then we can add 
return "false" if $attr == SAI_UDF_MATCH_ATTR_NEW_TYPE;
in front of line 2059.